doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+* There is now added support for NVMe/TCP (gateway groups, multiple
+  namespaces), multi-cluster management, oAuth2 integration, and enhanced
+  RGW/SMB features including multi-site automation, tiering, policies,
+  lifecycles, notifications, and granular replication.
+
+MGR
+---
+
+* Users now have the ability to force-disable always-on modules and the
+  removal of the restful and zabbix modules (both deprecated since 2020).
+  Note that the dashboard module's richer and better-maintained RESTful
+  API can be used as an alternative to the restful module, and the
+  prometheus module can be used as an alternative monitoring solution for
+  zabbix.
+
+
+RADOS
+-----
+
+* Long expected performance optimizations (FastEC) have been added for EC pools,
+  including partial reads and partial writes.
+* A new implementation of the Erasure Coding I/O code provides substantial performance
+  improvements and some capacity improvements. The new code is designed to optimize
+  performance when using Erasure Coding with block storage (RBD) and file storage
+  (CephFS) but will have some benefits for object (RGW) storage, in particular when
+  using smaller sized objects. A new flag ``allow_ec_optimizations`` needs to be set
+  on each pool to switch to using the new code. Existing pools can be upgraded once
+  the OSD and MON daemons have been updated. There is no need to update the clients.
+* The default erasure code plugin has been switched from Jerasure to ISA-L.
+* BlueStore now has better compression and a new, faster WAL (write-ahead-log).
+* All components have been switched to the faster OMAP iteration interface.
+* It is now possible to bypass ceph_assert()s in extreme cases to help with disaster
+  recovery.
+* mClock has received several bug fixes and and improved configuration defaults.
+* Testing improvements for dencoding verification were added.
+* Users can now track the data availability score for each pool in their cluster. This
+  feature is currently in tech preview. A pool is considered unavailable if any PG in
+  the pool is not in active state or if there are unfound objects. Otherwise the pool
+  is considered available. The score is updated every one second by default. The
+  feature is on by default.
+
+RBD
+---
+
+* New live migration features: RBD images can now be instantly imported
+  from another Ceph cluster (native format) or from a wide variety of
+  external sources/formats with the help of the new NBD stream and an
+  appropriately capable NBD server such as `qemu-nbd`.
+* There is now support for RBD namespace remapping while mirroring
+  between Ceph clusters.
+* New commands include `rbd group info` and `rbd group snap info`.
+* The `rbd group snap ls` command was enhanced.
+* The `rbd device map` command now defaults to msgr2.
+
+RGW
+---
+
+* Multiple fixes: Lua scripts will not run against health checks,
+  properly quoted ETag values returned by S3 CopyPart, PostObject and
+  CompleteMultipartUpload responses.
```

Comment:
Suggest, as I think this should be parsed:

*Lua scripts will no longer run uselessly against health checks
* `ETag`` values are now properly quoted in in S3 ``CopyPart``, ``PostObject`` and  ``CompleteMultipartUpload` responses.